frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteColorSettingsPane.java

```diff
@@ -0,0 +1,63 @@
+package com.virtuslab.gitmachete.frontend.file.highlighting;
+
+import java.util.Map;
+
+import javax.swing.Icon;
+
+import com.intellij.openapi.editor.colors.TextAttributesKey;
+import com.intellij.openapi.fileTypes.SyntaxHighlighter;
+import com.intellij.openapi.options.colors.AttributesDescriptor;
+import com.intellij.openapi.options.colors.ColorDescriptor;
+import com.intellij.openapi.options.colors.ColorSettingsPage;
+import icons.MacheteIcons;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.jetbrains.annotations.Nls;
+
+public class MacheteColorSettingsPane implements ColorSettingsPage {
+  private static final AttributesDescriptor[] DESCRIPTORS = new AttributesDescriptor[]{
```

Comment:
Here are defined user facing element names that will appear in color customization in settings and its default values